src/Plugins/DBFTPlugin/DBFTPlugin.json

```diff
@@ -7,5 +7,8 @@
     "MaxBlockSize": 2097152,
     "MaxBlockSystemFee": 150000000000,
     "UnhandledExceptionPolicy": "StopNode"
-  }
+  },
+  "Dependency": [
+    "StateService"
```

Comment:
@neo-project/core There's a pitfall that may be moved to a separate issue: dBFT needs StateService because dBFT should have an access to stateroots, and stateroots are managed by StateService plugin. However, StateService plugin requires RPCServer plugin to be installed because of RPC method extensions. So in other words, dBFT requires RPC server which is not a proper dependency for dBFT.

We may split StateService plugin functionality into two plugins: one part will manage stateroots, another part will expose additional RPC methods. Let me know if you agree or have other suggestions on how to solve this issue.